src/pages/project/test-big-project.md

```diff
@@ -0,0 +1,28 @@
+---
+title: '[Test] Big Project'
+summary: >-
```

Comment:
Pointing out here the default way that markdown widget fields get generated from Netlify CMS to `.md`.